src/content/docs/infrastructure/prometheus-integrations/integrations-list/node-exporter-integration.mdx

```diff
@@ -0,0 +1,418 @@
+---
+title: 'Node Exporter integration'
+tags:
+  - Integrations
+  - Node Exporter 
+  - Prometheus
+  - host monitoring 
+freshnessValidatedDate: 2024-04-10 
+---
+
+Do you want to monitor hardware and kernel metrics for a Linux server? You can do this with the New Relic remote write integration and the Prometheus Node Exporter. When you combine these two programs with the Prometheus monitoring system, you can send data to New Relic where you can analyze it. 
```

Comment:
The second sentence could use some more specificity--"where you can analyze it" feels a little open-ended and bland. Is there a way we can drill into the specifics of this?

Reply:
How about this: When you combine these two programs with the Prometheus monitoring system, you can send data to New Relic where you can use it for troubleshooting.

Reply:
I like the intro in the new build :)